lib/src/test/kotlin/network/account/AccountTest.kt

```diff
@@ -479,7 +479,7 @@ class AccountTest {
             nonce = Felt.ZERO,
             forFeeEstimate = true,
         )
-        assertEquals(payloadForFeeEstimation.version, Felt(BigInteger("340282366920938463463374607431768211457")))
+        assertEquals(Felt.fromHex("0x100000000000000000000000000000001"), payloadForFeeEstimation.version.value)
```

Comment:
nit:
* Why not `TransactionVersion.V1_QUERY`?
* Similar assert can be added to other `estimate fee` tests in this file